src/Providers/Transforms/PaymentConfiguration/S3PaymentConfigurationTransformDataProvider.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Amazon.S3;
+using form_builder.Extensions;
+using form_builder.Gateways;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+
+namespace form_builder.Providers.Transforms.PaymentConfiguration
+{
+    public class S3PaymentConfigurationTransformDataProvider : IPaymentConfigurationTransformDataProvider
+    {
+        private readonly IS3Gateway _s3Gateway;
+        private readonly IWebHostEnvironment _environment;
+        private readonly IConfiguration _configuration;
+
+        public S3PaymentConfigurationTransformDataProvider(IS3Gateway s3Service, IWebHostEnvironment environment, IConfiguration configuration)
+        {
+            _s3Gateway = s3Service;
+            _environment = environment;
+            _configuration = configuration;
+        }
+
```

Comment:
can we add associated tests for this transform, dont need one for the local transform.